Docs/Entities.md

```diff
@@ -0,0 +1,31 @@
+# Entities
+
+### User
+- ID ```number```
+- First Name ```string(25)```
+- Last Name ```string(25)```
+- Email ```string(100)```
+- Password ```string(100)```
+- Country ```string```
+
+### Skill
+- ID ```number```
+- Title ```string(50)```
+- Description ```string(5000)```
+- Type ```enum {Tech | non-Tech}```
+
+### Position
+- ID ```number```
```

Comment:
Please clarify the position, domain and roadmap. what is the difference ?

Reply:
Sure,

1. Domain : it's this general domain that's involve many positions (such as Web Development).
2. Position : it's the job position that user plan to work (such as Front-End and Back-End it's all in web development domain).
3. Roadmap : it's the ordered skills that user should take to achieve a specific position. _Keep in mind that each position may have more than one roadmap, but in our project we have one if nothing changed in out plan_.